user-interface/src/lib/components/Pill.tsx

```diff
@@ -39,7 +39,7 @@ export function Pill(props: PillProps) {
       onClick={() => props.onClick(props.value)}
       onKeyDown={(ev) => handleKeyDown(ev)}
       tabIndex={0}
-      aria-label={`${props.ariaLabelPrefix} - ${props.label} is currently selected.`}
+      aria-label={`${props.ariaLabelPrefix} - ${props.label} is currently selected. Press Enter key to unselect.`}
```

Comment:
Why do we need this additional verbiage? Space bar also works, so it seems like default behavior for a button.

Reply:
It is default interaction behavior, but the button removing itself isn't completely self explanatory. maybe something like `${props.ariaLabelPrefix} - ${props.label} filter. Select to remove` ?

Reply:
@shalliedragon

Reply:
I'm okay with having that. Though I'd say "deselect" instead of "unselect." Unselect isn't a word.

Reply:
@ShallieDragon, sorry, I just want to clarify. You're okay with having what, the current text or Arthur's suggestion. Note that both the enter and space bar keys perform the same action.

Reply:
I think it'd be good to have a programmatic indicator that says something like "click to deselect it". I think saying "click" specifically might be okay, because I think screen reader users know to interpret "click" as "press the interact keyboard button".